transforms/tabular/tabular-thresholding-plugin/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.9-slim
+
+COPY VERSION /
+		
+ARG EXEC_DIR="/opt/executables"
+ARG DATA_DIR="/data"
+
+ENV FILE_EXT="csv"
+
+RUN mkdir -p ${EXEC_DIR} \
+    && mkdir -p ${DATA_DIR}/inputs \
+    && mkdir ${DATA_DIR}/outputs
+
+COPY src ${EXEC_DIR}/
+WORKDIR ${EXEC_DIR}
+
+RUN apt-get -y update && \
+    apt-get -y install git && \
+    pip3 install -r ${EXEC_DIR}/requirements.txt --no-cache-dir --force && \
+    pip3 install git+https://github.com/nishaq503/thresholding.git@646f66f8c5f76317b8de7d16b918190eb7f3ec81
```

Comment:
I do not like depending on a package stored in a personal account. We need to move the code into the polus-plugins repository as long as it's not too lengthy. @nishaq503 if you would like to maintain credit in the git history, please open up a PR to Hamdahs branch that moves the thresholding code over.

Reply:
Thresholding function added in the repo